mono_repo/lib/src/commands/init.dart

```diff
@@ -5,15 +5,53 @@
 import 'dart:io';
 
 import 'package:args/command_runner.dart';
+import 'package:io/ansi.dart';
 import 'package:path/path.dart' as p;
 
 const _pubspecFileName = 'pubspec.yaml';
 const _pkgCfgFileName = 'mono_pkg.yaml';
 const _repoCfgFileName = 'mono_repo.yaml';
 const _recursiveScanFlag = 'recursive';
 
-const _commentText =
-    '# See with https://github.com/dart-lang/mono_repo for details on this file\n';
+const _repoCfgContents = r'''
+# See with https://github.com/dart-lang/mono_repo for details on this file
+self_validate: analyze_and_format
+
+github:
+  # Setting just `cron` keeps the defaults for `push` and `pull_request`
+  cron: '0 0 * * 0'
+  on_completion:
+    - name: "Notify failure"
+      runs-on: ubuntu-latest
+      # Run only if other jobs have failed and this is a push or scheduled build.
+      if: (github.event_name == 'push' || github.event_name == 'schedule') && failure()
+      steps:
+        - run: >
+            curl -H "Content-Type: application/json" -X POST -d \
+              "{'text':'Build failed! ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}'}" \
+              "${CHAT_WEBHOOK_URL}"
+          env:
+            CHAT_WEBHOOK_URL: ${{ secrets.BUILD_AND_TEST_TEAM_CHAT_WEBHOOK_URL }}
```

Comment:
This section should be commented out, as it won't work without some repo level configuration and it is also specific to the expected payload for google chat webhooks and won't for for slack, etc.

```suggestion
      # if: (github.event_name == 'push' || github.event_name == 'schedule') && failure()
      # steps:
      #   - run: >
      #       curl -H "Content-Type: application/json" -X POST -d \
      #       # Note that this is specific to google chat webhook format - other services expect
      #       # a different looking JSON payload.
      #         "{'text':'Build failed! ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}'}" \
      #         "${CHAT_WEBHOOK_URL}"
      #     env:
      #       CHAT_WEBHOOK_URL: ${{ secrets.<your-github-secret-id> }}
```